project.clj

```diff
@@ -6,7 +6,10 @@
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [org.onyxplatform/onyx "0.8.2"]]
   :plugins [[lein-cljfmt "0.3.0"]
-            [lein-cloverage "1.0.7-SNAPSHOT"]]
+            [lein-cloverage "1.0.7-SNAPSHOT"]
+            [lein-kibit "0.1.2"]
+            [lein-kibit "0.1.2"]
```

Comment:
This dependency is in there twice for reasons I'm going to assume is a side effect of this depending on an earlier PR?